addOns/graphql/src/main/java/org/zaproxy/addon/graphql/ExtensionGraphQl.java

```diff
@@ -165,6 +181,33 @@ private void stopParserThreads() {
         parserThreads.clear();
     }
 
+    private void addScript() throws IOException {
+        ExtensionScript extScript =
+                Control.getSingleton().getExtensionLoader().getExtension(ExtensionScript.class);
+        if (extScript != null) {
+            ScriptType variantType =
+                    extScript.getScriptType(ExtensionActiveScan.SCRIPT_TYPE_VARIANT);
+            ScriptEngineWrapper engine =
+                    extScript.getEngineWrapper(
+                            Constant.messages.getString("graphql.script.engine"));
+            if (variantType != null && engine != null) {
+                String scriptName = Constant.messages.getString("graphql.script.name");
```

Comment:
Might be better to hardcode, otherwise the user might end up with several scripts (e.g. when changing language).